test/system/user_registrations_test.rb

```diff
@@ -3,52 +3,9 @@
 require 'application_system_test_case'
 
 class UserRegistrationsTest < ApplicationSystemTestCase
-  test 'Blank inputs' do
-    visit new_user_registration_url
-
-    click_on 'Sign up'
-    assert page.has_content?("Email can't be blank")
-    assert page.has_content?("Password can't be blank")
-  end
-
-  test 'Existing account' do
-    User.create(email: 'existing@example.com', password: 'password1234')
-    visit new_user_registration_url
-
-    fill_in 'Email', with: 'existing@example.com'
-    click_on 'Sign up'
-    assert page.has_content?('Email has already been taken')
-  end
-
-  test 'Invalid password' do
-    visit new_user_registration_url
-
-    fill_in 'Email', with: 'test@example.com'
-    fill_in 'Password', with: '1234'
-
-    click_on 'Sign up'
-    assert page.has_content?('Password is too short (minimum is 6 characters)')
-  end
-
-  test 'Password confirmation' do
-    visit new_user_registration_url
-
-    fill_in 'Email', with: 'test@example.com'
-    fill_in 'Password', with: 'Password123'
-    fill_in 'Confirm Password', with: 'differentPassword'
-
-    click_on 'Sign up'
-    assert page.has_content?("Password confirmation doesn't match Password")
-  end
-
-  test 'Successful registration' do
-    visit new_user_registration_url
-
-    fill_in 'Email', with: 'test@example.com'
-    fill_in 'Password', with: 'Password123'
-    fill_in 'Confirm Password', with: 'Password123'
-
-    click_on 'Sign up'
-    assert page.has_content?('')
+  test 'user registration with GitHub' do
+    visit '/users/sign_up'
```

Comment:
Strange that the url helper function `new_user_registration_url` doesn't work here anymore 🤔 This is good for now, but we might want to figure out how to fix that at some point since it seems like some assertions aren't working either.

Reply:
When I debugged using the Rails console, it would seem to correctly print out the new_user_registration_url and new_user_registration_path, however instead of the localhost URL it had some strange code as the prefix, and both values were followed by a nil value.